packages/migrate/migrations/svelte-5/index.js

```diff
@@ -0,0 +1,175 @@
+import { resolve } from 'import-meta-resolve';
+import colors from 'kleur';
+import { execSync } from 'node:child_process';
+import fs from 'node:fs';
+import { dirname } from 'node:path';
+import { fileURLToPath, pathToFileURL } from 'node:url';
+import prompts from 'prompts';
+import semver from 'semver';
+import glob from 'tiny-glob/sync.js';
+import { bail, check_git, update_js_file } from '../../utils.js';
+import { migrate as migrate_svelte_4 } from '../svelte-4/index.js';
+import { transform_module_code, transform_svelte_code, update_pkg_json } from './migrate.js';
+
+export async function migrate() {
+	if (!fs.existsSync('package.json')) {
+		bail('Please re-run this script in a directory with a package.json');
+	}
+
+	const pkg = JSON.parse(fs.readFileSync('package.json', 'utf8'));
+	const svelte_dep = pkg.devDependencies?.svelte ?? pkg.dependencies?.svelte;
+	if (svelte_dep && semver.validRange(svelte_dep) && semver.gtr('4.0.0', svelte_dep)) {
+		console.log(
+			colors
+				.bold()
+				.yellow(
+					'\nDetected Svelte 3. We recommend running the `svelte-4` migration first (`npx svelte-migrate svelte-4`).\n'
+				)
+		);
+		const response = await prompts({
+			type: 'confirm',
+			name: 'value',
+			message: 'Run `svelte-4` migration now?',
+			initial: false
+		});
+		if (!response.value) {
+			process.exit(1);
+		} else {
+			await migrate_svelte_4();
+			console.log(
+				colors.bold().green('`svelte-4` migration complete. Continue with `svelte-5` migration?\n')
+			);
+			const response = await prompts({
+				type: 'confirm',
+				name: 'value',
+				message: 'Continue?',
+				initial: false
+			});
+			if (!response.value) {
+				process.exit(1);
+			}
+		}
+	}
+
+	let migrate;
+	try {
+		try {
+			({ migrate } = await import_from_cwd('svelte/compiler'));
+			if (!migrate) throw new Error('found Svelte 4');
+		} catch {
+			execSync('npm install svelte@5 --no-save', {
```

Comment:
this bombs out for me. should we make it `@next` so that folks can test the migration tool before a final release?
```suggestion
			execSync('npm install svelte@next --no-save', {
```